Integrations/integration-McAfee-MAR.yml

```diff
@@ -0,0 +1,332 @@
+description: Connect to MAR using its DXL client
+detaileddescription: |+
+  ### Prerequisites - Connect to McAfee Active Response (MAR) using the DXL MAR client.
+   - [Create certificates & configure dxl](https://opendxl.github.io/opendxl-client-python/pydoc/index.html). After this phase you must have:
+       * Broker CA certificates (`brokercerts.crt` file)
+       * Client certificate (`client.crt` file)
+       * Client private key (`client.key` file)
+       * Broker list properties file (`brokerlist.properties` file)
+
+   - [Enable MAR Search API](https://opendxl.github.io/opendxl-client-python/pydoc/marsendauth.html)
+
+  ### API Documentation:
+  - MAR docs - [Here](https://kc.mcafee.com/resources/sites/MCAFEE/content/live/PRODUCT_DOCUMENTATION/26000/PD26820/en_US/mar_200_pg_0-00_en-us.pdf)
+
+  ### Dependencies (Python packages) - No need to install, already installed in docker image :
+  dxlclient [docs](https://opendxl.github.io/opendxl-client-python/pydoc/index.html)
+  dxlmarclient [docs](https://opendxl.github.io/opendxl-mar-client-python/pydoc/)
+
+configuration:
+- display: Broker CA certificates content (see `brokercerts.crt` in instructions)
+  name: broker_ca_bundle
+  defaultvalue: ""
+  type: 12
+  required: true
+- display: Client certificates content (see `client.crt` in instructions)
+  name: cert_file
+  defaultvalue: ""
+  type: 12
+  required: true
+- display: Client private key content (see `client.key` in instructions)
+  name: private_key
+  defaultvalue: ""
+  type: 12
```

Comment:
you may want to change this to type 14
Which is ConfigDataTypeTextAreaEncrypted